.github/checkgroup.yml

```diff
@@ -0,0 +1,134 @@
+custom_service_name: "Lightning CI required checker"
+subprojects:
+  - id: "pytorch_lightning"
+    paths:
+      - ".circleci/**"
```

Comment:
Maybe also as an additional split, if we change something only in GPU test pipeline, we do not need to run all CPU testing, right?